source/StartConditions.cpp

```diff
@@ -13,15 +13,24 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #include "StartConditions.h"
 
 #include "DataNode.h"
+#include "DataWriter.h"
 #include "GameData.h"
 #include "Planet.h"
 #include "Ship.h"
+#include "SpriteSet.h"
 #include "System.h"
 
 using namespace std;
 
 
 
+StartConditions::StartConditions(const DataNode &node)
+{
+	Load(node);
+}
+
+
+
 void StartConditions::Load(const DataNode &node)
 {
 	for(const DataNode &child : node)
```

Comment:
@tehhowch I think we shouldn't do this because it would cause inconsistencies between named and anonymous start scenarios, in the former the `name` statement wouldn't be necessary and in the latter one it would. Unless you mean after loading, in case `name` was never set?

Reply:
Think of it like missions.
```
mission <name>
	name <name>
```
Missions have a unique name and a display name. If no display name is given, the unique name is displayed in game. So while having a separate display name for a start that is already named isn't necessary, that doesn't mean it should be disallowed. One might want to uniquely name starts separately from the display names. (And I am such one.)